docs/developer/app-store/apps/taxes.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Taxes
+sidebar_position: 2
+---
+
+# Taxes
+
+## Overview
+
+_Taxes_ is a Saleor app that allows delegating tax calculations to external tax providers. Here are the currently available providers:
+
+- [TaxJar](https://www.taxjar.com/)
+- [Avatax](https://www.avalara.com/us/en/products/calculations.html)
+- [Avatax Excise](https://developer.avalara.com/excise/) [_coming soon_]
+
+Taxes App has two views: **Channels** and **Providers**.
+
+**Channels** allows you to fill in the "ship from" address and assign a tax provider instance to the channel.
+
+**Providers** allows you to manage the tax provider instances. You can have multiple instances of the same provider.
+
+:::caution
+
+To successfully configure the Taxes App, you must have an account with one of the supported tax providers.
+
+:::
```

Comment:
Maybe we can have a "prerequisites" section for every app/readme?

Reply:
I thought about it, and decided to handle it this way.

Would you like a "Prerequisites" section with the same content as this `caution` element, or would you like to have more information there?

Reply:
I think it should be standard section for every app, but content will be different, depending on apps needs